src/test/rgw/test_rgw_iam_policy.cc

```diff
@@ -367,6 +367,9 @@ TEST_F(PolicyTest, Parse3) {
   act2[s3GetReplicationConfiguration] = 1;
   act2[s3GetObjectTagging] = 1;
   act2[s3GetObjectVersionTagging] = 1;
+  act2[s3GetBucketObjectLockConfiguration] = 1;
+  act2[s3GetObjectRetention] = 1;
+  act2[s3GetObjectLegalHold] = 1;
```

Comment:
these are failing to compile with:
> src/test/rgw/test_rgw_iam_policy.cc:370:8: error: 's3GetBucketObjectLockConfiguration' was not declared in this scope
   act2[s3GetBucketObjectLockConfiguration] = 1;

this is because the name is in namespace rgw::IAM. the other tags are imported directly with:
> using rgw::IAM::s3GetObjectVersionTagging;